scripts/convex-to-d1-migration.ts

```
@@ -0,0 +1,1535 @@
+      columns = [
+        'id',
+        'letter_id',
+        'owner_id',
+        'scheduled_at',
+        'status',
+        'attempt_count',
+        'last_attempt_at',
+        'created_at',
+      ]
+      enumValue(document, 'status', DELIVERY_STATUSES, table.source)
+      const scheduledAt = timestamp(document, 'scheduledAt')
```

Biome (2.5.8)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)